apps/web-giddh/src/app/shared/header/components/account-add-new-details/account-add-new-details.component.ts

```diff
@@ -195,7 +195,8 @@ export class AccountAddNewDetailsComponent implements OnInit, OnChanges, AfterVi
         this.initializeNewForm();
         this.activeGroup$.subscribe(response => {
             if (response) {
-                if (response.parentGroups && response.parentGroups.length) {
+           if (response.parentGroups && response.parentGroups.length
```

Comment:
Consider formatting this `if` block with `Format this selection` option in VS code (Select the code block > Right click > Choose option `Format this selection`)